### PR TITLE
Add send txn examples to react templates

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: blank_rust_react_project/src/App.js [hunk 1]
```diff
@@ -53,6 +59,12 @@ class App extends Component {
     console.log("after sign out", this.props.wallet.isSignedIn())
   }
 
+  async changeGreeting() {
+    console.log(this);
```
Comment (vgrichina) [inline blank_rust_react_project/src/App.js]: is this on purpose or accidental?

File: blank_react_project/src/App.js [hunk 2]
```diff
@@ -53,6 +59,12 @@ class App extends Component {
     console.log("after sign out", this.props.wallet.isSignedIn())
   }
 
+  async changeGreeting() {
+    console.log(this);
```
Comment (vgrichina) [inline blank_react_project/src/App.js]: is this on purpose or accidental?

File: blank_react_project/src/App.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ class App extends Component {
     this.requestSignIn = this.requestSignIn.bind(this);
     this.requestSignOut = this.requestSignOut.bind(this);
     this.signedOutFlow = this.signedOutFlow.bind(this);
+    this.changeGreeting = this.changeGreeting.bind(this);
```
Comment (vgrichina) [inline blank_react_project/src/App.js]: why do we need these? That doesn't look right, we don't do such stuff in NEAR Wallet.

File: blank_rust_react_project/src/App.js (hunk 1 above)
Comment (janedegtiareva) [reply blank_rust_react_project/src/App.js]: accidental. thanks!

File: blank_react_project/src/App.js (hunk 2 above)
Comment (janedegtiareva) [reply blank_react_project/src/App.js]: same here

File: blank_react_project/src/App.js (hunk 3 above)
Comment (janedegtiareva) [reply blank_react_project/src/App.js]: this is null inside changeGreeting unless this line is present.

**Pull request merged**